### ISSUE TITLE
Impossible to continue using F# Interactive (FSI) if you miss-escape a string

### ISSUE THREAD

**Issue by abelbraaksma** [Microsoft/visualfsharp#4802]
Try something like the following line in F# Interactive (`System` must be open):

```f#
 Uri.IsWellFormedUriString ("c:\", UriKind.Absolute);;
```

### Repro steps

See above.

### Expected behavior

Somehow being able to continue using F# Interactive, with Ctrl-C, or typing more ";;" or more enters.

### Actual behavior

I didn't quite find a way to continue. This is probably expected behavior (the string is unfinished), but it would be nice to have a way to cancel your current input, whatever it is.

### Known workarounds

Find a way to close the quote. In this case, once I realized the error after not understanding the blank lines I got after hitting enter, I typed the `"` (double quote char) and I got my console back.

**Comment by dsyme** [comment]
You have to send-to-interactive just the `"` character.  

I do agree this should be improved.

**Comment by abelbraaksma** [comment]
Yes, but since there's no syntax highlighting, this is more baffling than it should be
